myl.py

```diff
@@ -59,7 +60,13 @@ def mail_to_dict(msg, date_format="%Y-%m-%d %H:%M:%S"):
             "html": msg.html,
             "text": msg.text,
         },
-        "attachments": msg.attachments,
+        "attachments": [
+            {
+                "filename": x.filename,
+                "content": base64.b64encode(x.payload).decode("utf-8"),
```

Comment:
It might be good to save the MIME content-type here also, from [RFC 2045](https://www.rfc-editor.org/rfc/rfc2045#section-5).